### PR TITLE
Portfolio: contact form added and mobile version finished

### PR DESCRIPTION
**Pull request by Alexr16**
Contact form added.
Fields validation form added.
Forms Free is integrated to collect contact form data.

Note: 

- Message field accepts up to 500 chars.
- Name field accepts up to 30 chars.
- Email field accepts only emails.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -221,6 +221,50 @@ <h2>About me</h2>
...
+    <section class="portfolio-form">
+        <div class="form-text">
+            <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        </div>
+        <div class="form-area">
+            <form action="https://formspree.io/f/xzboreoy" method="post">
+                <ul class="contact-form">
+                    <li class="name-input">
+                        <input class="names" type="text" name="First Name" maxlength="30" placeholder="First Name">
+                        <input class="names" type="text" name="Last Name" maxlength="30" placeholder="Second Name">
```
Comment (RolandM99) [inline index.html]: - [ ] From the template you choose to work on, there is no input for `first name` and` last name`. there is just one input for the `full name`. To make your code DRY and KISS, kindly remove those two inputs (first name and last name).

File: index.html [hunk 2]
```diff
@@ -221,6 +221,50 @@ <h2>About me</h2>
...
+                    </li>
+                    <li class="full-name-input">
+                        <input class="full-name" type="text" name="name" placeholder="Full name">
```
Comment (RolandM99) [inline index.html]: - [ ] Please, Kindly add the `maxlength="30" ` attribute to the name input

File: index.html [hunk 3]
```diff
@@ -221,6 +221,50 @@ <h2>About me</h2>
...
+                    </li>
+                    <li>
+                        <input class="email" type="email" name="Email address" placeholder="Email address">
```
Comment (RolandM99) [inline index.html]: - [ ] All input fields must have the `required `attribute to enable validation on submission. let's go, please kindly add them🧑‍🔧
- Example : 
```
<input class="email" type="email" name="Email address" placeholder="Email address" required >
```

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi  @Alexr16 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
 - No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alexr16** [comment]
Inputs `first name` and `last name` removed.
`maxlength="30"` attribute added to the full name input.
`required` attribute added to all the inputs file.

**Review by CollinsTatang — APPROVED** [review]
Hi @Alexr16 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**